### DISCUSSION TITLE
💡 Feature Request: Display the floor next to the "<playername>'s Dungeon"

### DISCUSSION THREAD

**Discussion by Dx8pi** [DiscordDungeons/Bugs#1334] [Ideas]
### Is your feature request related to a problem?

Not really, but you can't see the floor you're on unless you remembered it when entering that floor.

### Describe alternatives you've considered

Like this

Dx8pi's Dungeon - Floor 3

only way to see it currently is by descending and remembering it before moving.

**Comment by Willard21** [comment]
`/dungeon status` is a thing

**Comment by Dx8pi** [comment]
Yes, but that clogs up the chat, so you eventually have to scroll up to press the buttons. Ideally I would want the dungeon board at the bottom of all the messages as often as possible. I believe other players would agree with me. Besides, it's not that difficult of an addition to make, no?
